crates/wash-cli/src/cmd/dev/session.rs

```diff
@@ -299,7 +299,11 @@ impl WashDevSession {
                     wadm_log_path.display()
                 )
             })?;
-        let wadm_binary = ensure_wadm(&wadm_opts.wadm_version, &install_dir).await?;
+        let wadm_binary = ensure_wadm(
+            &wadm_opts.wadm_version.unwrap_or(WADM_VERSION.to_owned()),
```

Comment:
Good question: since the downloaded binaries are a bit different

```console
ls ~/.wash/downloads/
```

results as 

```
nats-server*   nats.log       v1.4.2/        wadm.log       wasmcloud.log
nats.conf      nats.pid       wadm*          wadm.pid       wasmcloud.pid
```
(where the `v1.4.2` is the wasmCloud binary containing folder).

or similar. Currently only the wasmCloud binary has its own folder, everything else is "just" in the folder. That is the reason for the difference here.

Should I create subfolders for the binaries?

Reply:
Oh, nvm this is unrelated, will update in the morning.